### ISSUE TITLE
Admin Reputation Tools and Testing

### ISSUE THREAD

**Issue by danielBingham** [danielBingham/peerreview#148]
## Story Description 
*What are we trying to accomplish in plain english?*

We need admin tools for generating and managing reputation.  These will mostly be used for testing.

## Acceptance Criteria
*At what point is this story considered "done"?*

- [ ] We have an admin dashboard where we can generate reputation for a user using:
    - [ ] an OpenAlex ID
    - [ ] An ORCID ID
- [ ] We can retrigger reputation generation for a user.
- [x] On Staging, we can trigger reputation generation from an ORCID ID by simply assigning the ORCID ID.  This process uses the same end flow as ORCID registration or connection and allows us to test it.
- [x] We have an admin endpoint allowing a user's reputation and orcid id to be reset.

## Dependencies
*What stories does this one depend on?  What do we need to do first before we can call this one done?*

**Comment by danielBingham** [comment]
- [ ] We can wipe out a user's orcid and reputation.
- [x] Users can set their own orcid Id to trigger the reputation generation flow for testing.

**Comment by danielBingham** [comment]
Going to defer the admin dashboard pieces of it.  The API gives us enough for now.

**Comment by danielBingham** [comment]
Admin dashboard deferred to #160
